### PR TITLE
Ballot SC27

### PR DESCRIPTION
**Pull request by wthayer**

### CODE REVIEW

**Comment by wthayer** [comment]
Be sure to "squash & merge" to remove the extraneous commits used to synch the ballot branch with cabforum/master

File: docs/BR.md [hunk 1]
```diff
@@ -122,6 +122,8 @@ The following Certificate Policy identifiers are reserved for use by CAs as an o
 | 1.6.7 | SC23 | Precertificates | 14-Nov-2019	| 19-Dec-2019 |
 | 1.6.7 | SC24 | Fall Cleanup v2 | 12-Nov-2019	| 19-Dec-2019 |
 | 1.6.8 | SC25 | Define New HTTP Domain Validation Methods v2 | 31-Jan-2020	| 3-Mar-2020 |
+| 1.6.9 | SC27 | Version 3 Onion Certificates v3 | 19-Feb-2020	| 27-Mar-2020 |
```
Comment (dzacharo) [inline docs/BR.md:125]: This should be "Version 3 Onion Certificates" instead of "Version 3 Onion Certificates v3"

File: docs/EVG.md [hunk 2]
```diff
@@ -76,6 +76,7 @@ The CA/Browser Forum is a voluntary open organization of certification authoriti
 | 1.6.9 | SC16 | Other Subject Attributes | 15 Mar 2019 | 16 Apr 2019 |
 | 1.7.0 | SC17 | Alternative registration numbers for EV certificates | 21 May 2019 | 21 June 2019 |
 | 1.7.1 | SC24 | Fall cleanup v2 | 12 Nov 2019 | 19 Dec 2019 |
+| 1.7.2 | SC27 | Version 3 Onion Certificates v3 | 19-Feb-2020	| 27-Mar-2020 |
```
Comment (dzacharo) [inline docs/EVG.md:80]: This should be "Version 3 Onion Certificates" instead of "Version 3 Onion Certificates v3"

**Comment by dzacharo** [comment]
@wthayer see comments above. The rest of the changes LGTM.

File: docs/BR.md (hunk 1 above)
Comment (wthayer) [reply docs/BR.md]: I removed "v3" from the version table, but I will note that the version number of other ballots has been included in that table

**Comment by wthayer** [comment]
@dzacharo please review again.

**Pull request merged**